intelmq/Dockerfile

```diff
@@ -15,6 +15,12 @@ RUN apt-get update && apt-get install -y jq git \
     python3-pip python3-psycopg2 \
     python3-bs4 sudo libgpgme-dev swig sqlite3
 
+# pip installs some packages (intelmq-api, intelmq-manager) into /usr/lib/python3.8/site-packages
+# instead of /usr/local/lib/python3.10/dist-packages (which is the default location, also in python's sitepath)
+# https://github.com/Intevation/intelmq-cb-mailgen-docker/issues/9
+# https://github.com/pypa/pip/issues/10805
+RUN ln -s /usr/local/lib/python3.8/dist-packages/ /usr/lib/python3.8/site-packages
```

Comment:
I wonder if this is TRTTD?  Isn't possible to force an valid installation prefix path at installation time?
(a short search makes me think the `--target` option of pip might be worth a look)

Reply:
Hmm, ok, then I'll merge for now, although I'm really unhappy with this workaround.  We should check back with some
Python/pip experts there has to be a clean solution (or at least I hope so)...

Reply:
AFAIK the whole dist-packages thing is Debian-/Ubuntu-specifc. upstream pip behaves differently (and as expected)